tests/logprob/test_order.py

```diff
@@ -254,3 +254,26 @@ def test_max_discrete(mu, size, value, axis):
         (x_max_logprob.eval({x_max_value: test_value})),
         rtol=1e-06,
     )
+
+
+@pytest.mark.parametrize(
+    "mu, size, value, axis",
+    [(2, 3, 1, -1), (2, 3, 1, 0), (1, 2, 2, None), (0, 4, 0, 0)],
+)
+def test_min_discrete(mu, size, value, axis):
+    x = pm.Poisson.dist(name="x", mu=mu, size=(size))
+    x_min = pt.min(x, axis=axis)
+    x_min_value = pt.vector("x_min_value")
+    x_min_logprob = logp(x_min, x_min_value)
+
+    test_value = [value]
+
+    n = size
+    exp_rv = sp.poisson(mu).cdf(test_value[0]) ** n
+    exp_rv_prev = sp.poisson(mu).cdf(test_value[0] - 1) ** n
+
+    np.testing.assert_allclose(
+        (np.log(exp_rv_prev - exp_rv)),
+        (x_min_logprob.eval({x_min_value: (test_value)})),
+        rtol=1e-06,
+    )
```

Comment:
What is being asserted is -inf == -inf, which is maybe why we are noticing that this test is passing. Change wrt formula